rpc/core/consensus.go

```diff
@@ -7,9 +7,19 @@ import (
 	"github.com/tendermint/tendermint/types"
 )
 
-func Validators() (*ctypes.ResultValidators, error) {
-	blockHeight, validators := consensusState.GetValidators()
-	return &ctypes.ResultValidators{blockHeight, validators}, nil
+func Validators(heightPtr *int) (*ctypes.ResultValidators, error) {
+	if heightPtr == nil {
```

Comment:
Could we add a comment here.

I am assuming that if `heightPtr == nil` it means that we get the latest validators and that's the reason why this cannot throw an error.

Reply:
Added to the docustring